src/content/edit-user-script.html

```diff
@@ -39,6 +40,36 @@
 </head>
 
 <body>
+<div id="modal">
+  <div class="panel">
```

Comment:
Why do I need an extra wrapper div here?

Reply:
The outer div covers the entire page and consists of a semi transparent dark color. Effectively "dimming" the contents. The inner div is the focused "box". If you have suggestion on a better way to do this, shoot.